doc/pylint.rst

```diff
@@ -8,7 +8,8 @@ Spyder's code analyzer is powered by the best-in-class `Pylint`_ back-end, which
 
 .. _Pylint: https://www.pylint.org/
 
-.. image:: images/pylint/code_analysis_standard.png
+.. image:: images/pylint/code-analysis-standard.png
```

Comment:
Would be nice to make this full-width since it almost is already and it is the hero, but if it would take more than 5 minutes to redo not worth it I guess.